tests/SprykerTest/Client/RabbitMq/Model/Connection/ConnectionBuilderTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerTest\Client\RabbitMq\Model\Connection;
+
+use Codeception\Test\Unit;
+use Generated\Shared\Transfer\StoreTransfer;
+use PhpAmqpLib\Exception\AMQPIOException;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionBuilder\ConnectionBuilder;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionBuilder\ConnectionBuilderInterface;
+use Spryker\Client\RabbitMq\Model\Helper\QueueEstablishmentHelperInterface;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerTest
+ * @group Client
+ * @group RabbitMq
+ * @group Model
+ * @group Connection
+ * @group ConnectionBuilderTest
+ * Add your own group annotations below this line
+ */
+class ConnectionBuilderTest extends Unit
+{
+    /**
+     * @var string
+     */
+    protected const STORE_NAME = 'DE';
+
+    /**
+     * @var string
+     */
+    protected const LOCALE_CODE = 'en_US';
+
+    /**
+     * @var string
+     */
+    protected const QUEUE_POOL_NAME = 'synchronizationPool';
+
+    /**
+     * @var string
+     */
+    protected const INCORRECT_LOCALE_CODE = 'INCORRECT_LOCALE_CODE';
+
+    /**
+     * @var string
+     */
+    protected const INCORRECT_CONNECTION_NAME = 'INCORRECT_CONNECTION_NAME';
+
+    /**
+     * @var \SprykerTest\Client\RabbitMq\RabbitMqClientTester
+     */
+    protected $tester;
+
+    /**
+     * @return void
+     */
+    public function testCreateConnectionByQueueConnectionTransfer(): void
+    {
+        // Act
+        $this->createConnectionBuilder()->createConnectionByQueueConnectionTransfer($this->tester->createQueueConnectionTransfer());
+    }
+
+    /**
+     * @return void
+     */
+    public function testcreateConnectionsByQueueConnectionTransfers(): void
+    {
+        // Act
+        $this->createConnectionBuilder()->createConnectionsByQueueConnectionTransfers([$this->tester->createQueueConnectionTransfer()]);
```

Comment:
Please re-arrange to have `// Arrange //Act // Assert` annotations inside the main method.